liberty-maven-plugin/src/it/tests/test-server-param-default/src/test/java/net/wasdev/wlp/maven/test/app/PluginConfigXmlIT.java

```diff
@@ -0,0 +1,108 @@
+        File f1 = new File(SOURCE_BOOTSTRAP_PROPERTIES);
+        File f2 = new File(nodeValue);
+        Assert.assertEquals("bootstrapPropertiesFile value", f1.getAbsolutePath(), f2.getAbsolutePath());
+        Assert.assertEquals("verify target server bootstrap.properties", FileUtils.fileRead(f2),
+                FileUtils.fileRead(TARGET_BOOTSTRAP_PROPERTIES));
+    }
+    
+    @Test
+    public void testJvmOptionsFileElements() throws Exception {
+        File in = new File(CONFIG_XML);
+        FileInputStream input = new FileInputStream(in);
+        
+        // get input XML Document
+        DocumentBuilderFactory inputBuilderFactory = DocumentBuilderFactory.newInstance();
+        inputBuilderFactory.setIgnoringComments(true);
+        inputBuilderFactory.setCoalescing(true);
+        inputBuilderFactory.setIgnoringElementContentWhitespace(true);
+        inputBuilderFactory.setValidating(false);
+        DocumentBuilder inputBuilder = inputBuilderFactory.newDocumentBuilder();
+        Document inputDoc = inputBuilder.parse(input);
+        
+        // parse input XML Document
+        XPath xPath = XPathFactory.newInstance().newXPath();
+        String expression = "/liberty-plugin-config/jvmOptionsFile/text()";
+        String nodeValue = (String) xPath.compile(expression).evaluate(inputDoc, XPathConstants.STRING);
+        File f1 = new File(SOURCE_JVM_OPTIONS);
+        File f2 = new File(nodeValue);
+        Assert.assertEquals("jvmOptionsFile value", f1.getAbsolutePath(), f2.getAbsolutePath());
+        Assert.assertEquals("verify target server jvm.options", FileUtils.fileRead(f2),
+                FileUtils.fileRead(TARGET_JVM_OPTIONS));
+    }
+    
+    @Test
+    public void testServerEnvElements() throws Exception {
+        File in = new File(CONFIG_XML);
+        FileInputStream input = new FileInputStream(in);
+        
+        // get input XML Document
+        DocumentBuilderFactory inputBuilderFactory = DocumentBuilderFactory.newInstance();
+        inputBuilderFactory.setIgnoringComments(true);
+        inputBuilderFactory.setCoalescing(true);
+        inputBuilderFactory.setIgnoringElementContentWhitespace(true);
+        inputBuilderFactory.setValidating(false);
+        DocumentBuilder inputBuilder = inputBuilderFactory.newDocumentBuilder();
+        Document inputDoc = inputBuilder.parse(input);
+        
+        // parse input XML Document
+        XPath xPath = XPathFactory.newInstance().newXPath();
+        String expression = "/liberty-plugin-config/serverEnv/text()";
+        String nodeValue = (String) xPath.compile(expression).evaluate(inputDoc, XPathConstants.STRING);
+        File f1 = new File(SOURCE_SERVER_ENV);
+        File f2 = new File(nodeValue);
+        Assert.assertEquals("serverEnv value", f1.getAbsolutePath(), f2.getAbsolutePath());
+        Assert.assertEquals("verify target server server.env", FileUtils.fileRead(f2),
```

Comment:
If you use import static org.junit.Assert.*; you can reference static methods on the Assert class without specifying the class in which the field is defined. I think it makes for cleaner test code